### DISCUSSION TITLE
Side Buttons from the right side of my mouse?

### DISCUSSION THREAD

**Discussion by scaraux** [noah-nuebling/mac-mouse-fix#124] [Questions] [answered]
I have a SteelSeries mouse with four buttons:

The two on the left are `button4` and `button5` and the two on the right are `page_up` and `page_down`.

The mouse fix app detects the Side Buttons from the **left side**.
I would like to do the opposite. I would like to "Move Between Spaces" using my right buttons.

How can I achieve this?

**Comment by noah-nuebling** [comment]
[SteerMouse](https://plentycom.jp/en/steermouse/) should work for that. 
SteerMouse is a great alternative to Mac Mouse Fix.
SteerMouse has custom code to support tons of devices, even devices that send keyboard events instead of normal mouse events like yours. I'm not sure, but it's very likely, that SteerMouse can properly handle your side buttons.

I'd love to fully support mice with buttons that send weird events like that in the future, too. But that's quite hard, so I can't give you a timeframe, unfortunately.

**Comment by noah-nuebling** [comment, marked as answer]
Hi! Marking this as answered since it's been transferred to issue #351. Please continue any discussion there.
